adafruit_veml6075.py

```diff
@@ -33,6 +33,12 @@
 from adafruit_bus_device.i2c_device import I2CDevice
 from micropython import const
 
+try:
+    from typing import Union  # pylint: disable=unused-import
```

Comment:
`Union` isn't being used, so:

```suggestion
    import typing  # pylint: disable=unused-import
```